### PR TITLE
 Add Scheduler ResourceManager Powershell.

### PR DESCRIPTION
**Pull request by Sumanan**
This change add following cmdlets for RM Powershell:

 1)  Disable-AzureRmSchedulerJobCollection
 2)  Enable-AzureRmSchedulerJobCollection
 3)  Get-AzureRmSchedulerJobCollection
 4)  New-AzureRmSchedulerJobCollection
 5)  Remove-AzureRmSchedulerJobCollection
 6)  Set-AzureRmSchedulerJobCollection
 7)  Get-AzureRmSchedulerJob
 8)  Get-AzureRmSchedulerJobHistory
 9)  New-AzureRmSchedulerHttpJob
 10) New-AzureRmSchedulerServiceBusQueueJob
 11) New-AzureRmSchedulerServiceBusTopicJob
 12) New-AzureSchedulerStorageQueueJob
 13) Remove-AzureSchedulerJob
 14) Set-AzureRmSchedulerHttpJob
 15) Set-AzureRmSchedulerServiceBusQueueJob
 16) Set-AzureRmSchedulerServiceBusTopicJob
 17) Set-AzureSchedulerStorageQueueJob

 Missing from this Change:
 Support for Advance schedule and Retry policy. Will be part of next release

Fix.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Sumanan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by Sumanan** [comment]
Hi,

Build is failing due to missing file (System.Management.Automation.dll) in ExpressRoute project. I haven’t made any changes to ExpressRoute project and it builds fine locally (via msbuild build.proj). Could you please help me resolve this issue? We don’t want to miss the August train.

15:30:34
15:30:34 "D:\workspace\powershell\build.proj" (Full target) (1) ->
15:30:34 "D:\workspace\powershell\setup\azurepowershell.sln" (Build target) (179) ->
15:30:34 "D:\workspace\powershell\setup\azurepowershell.wixproj.metaproj" (default target) (180) ->
15:30:34 "D:\workspace\powershell\setup\azurepowershell.wixproj" (default target) (182) ->
15:30:34 (Link target) ->
15:30:34   D:\workspace\powershell\setup\azurecmdfiles.wxi(3872): error LGHT0103: The system cannot find the file 'D:\workspace\powershell\setup\..\src\Package\Debug\ServiceManagement\Azure\ExpressRoute\System.Management.Automation.dll'. [D:\workspace\powershell\setup\azurepowershell.wixproj]


Full Log: http://azuresdkci.cloudapp.net/job/powershell/5959/console
PR: https://github.com/Azure/azure-powershell/pull/2663

Thanks,
Sumanan Jeya

File: src/ResourceManager/AzureBatch/Commands.Batch/Accounts/GetBatchAccountCommand.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// ----------------------------------------------------------------------------------
```
Comment (markcowl) [inline src/ResourceManager/AzureBatch/Commands.Batch/Accounts/GetBatchAccountCommand.cs]: There should be no changes outside your library.  Please rebase.

**Comment by markcowl** [comment]
@Sumanan Please rebase, you have many changes to libraries outside yours which you should not.

**Pull request closed without merging**